### DISCUSSION TITLE
The importer could not connect to Firefly --- Port in FIREFLY_III_URL is ignored?

### DISCUSSION THREAD

**Discussion by Kirmy34** [firefly-iii/firefly-iii#5374] [Firefly III]
So I am running Firefly in a docker container accesable through port 8099 and I want to start using the Data Importer from a different container. After followig [the instructions](https://docs.firefly-iii.org/data-importer/install/configure/)  I cannot get the importer container connect to the firefly API...

The importer could not connect to Firefly III. Please remedy the error below first, and check out the documentation if necessary.

`Error accessing "raspi4/api/v1/about". Status code is 404. Body is: <html><head><title>404 Not Found</title></head><body> <h1>Not Found</h1> The URL you requested (/api/v1/about) was not found. <hr> Generated by darkhttpd/1.12 on Wed, 08 Dec 2021 20:36:00 GMT </body></html>`

The  return message is the default 404 page from a different service running on the host on port 80.

The docker run command looks as follows:

`docker run --rm \
--name=$SERVICE \
-e FIREFLY_III_ACCESS_TOKEN=$FF_TOKEN \
-e FIREFLY_III_URL=$FF_URL \
-e NORDIGEN_ID=$NDG_ID \
-e NORDIGEN_SECRET=$NDG_SECRET \
-p 8098:8080 \
fireflyiii/data-importer:latest`

where $FF_URL=raspi4:8099

but for some reason the port gets ignored?

After pressing Reauthenticate at the bottom of the page, it seems to work fine...
I didn't think this was serious enough for a bug report so here we are :D

**Comment by JC5** [comment]
Thanks for letting me know!

The port is never ignored in the URL, but I see you omitted the "http://" or "https://" part. In that case, your milage may vary.

**Reply by Kirmy34** [reply]
I also tried it with "http://", same deal, but thanks for the quick response

**Reply by JC5** [reply]
I hope to fix this in the next release regardless, a lot of this code is updated.
